content/en/providers/cloud-compute/_index.md

```diff
@@ -5,44 +5,42 @@ weight: 30
 type: "docs"
 ---
 
-This documentation covers how to join the EGI Cloud federation as a provider.
-If you are interested in joining please first contact EGI operations team
-(``operations _at_ egi.eu``), expressing interest and providing few details about:
+This documentation covers how to join the EGI Cloud federation as a provider. If
+you are interested in joining please first contact EGI operations team
+(`operations _at_ egi.eu`), expressing interest and providing few details about:
```

Comment:
So at various places we have real email, like for the support page where we have:
```markdown
[support@egi.eu](mailto:support@egi.eu)
```
We probably want to have a coherent way of dealing with this.
I'm quite fine with having the real emails, they are probably already here and there on the net.

Reply:
Fixed